### PR TITLE
Listing surveys to student role

### LINKED ISSUE #450: List Surveys
List surveys so student can choose which one to respond

### PR DESCRIPTION
**Pull request by amiapmorais**
fix #450 

- [ ] **Model testing.**
- [ ] **Request testing.**
- [ ] **System testing.**
- [x] **No tests required.**

### CODE REVIEW

**Comment by amiapmorais** [comment]
Code climate needs a method have at most 40 lines of code. Seriously?

**Comment by edimossilva** [comment]
> Code climate needs a method have at most 40 lines of code. Seriously?

Lol, https://medium.com/oril/10-tips-to-keep-your-code-more-clean-2fa9aafea1cf

File: android-client/Espoo/app/src/main/res/layout/survey_item_view.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp">
+
+    <ImageView
+        android:id="@+id/imageView3"
```
Comment (edimossilva) [inline android-client/Espoo/app/src/main/res/layout/survey_item_view.xml:10]: Is it ok to use ids like this? Or should you rename them?

**Review by edimossilva — COMMENTED** [review]
What are your plans for code climate issue?
You can config it to ignore the method if you think it is a good choice

**Comment by amiapmorais** [comment]
> What are your plans for code climate issue? You can config it to ignore the method if you think it is a good choice

I'm working on codeclimate issues. Sorry for not telling you. I'll improve my communication skills.

File: android-client/Espoo/app/src/main/res/layout/survey_item_view.xml (hunk 1 above)
Comment (amiapmorais) [reply android-client/Espoo/app/src/main/res/layout/survey_item_view.xml:10]: That is the android studio generated ids. I forgot to rename them. I'll do it.

**Pull request merged**